main: release 0.16.0

aykevl opened the pull request.

aykevl committed:
main: release 0.16.0

QuLogic commented on CHANGELOG.md:
```diff
@@ -1,3 +1,43 @@
+  - `test`: add support for -c and -o flags
+  - `test`: implement some benchmark stubs
+* **compiler**
+  - `builder`: improve detection of clang on Fedora
```
Can we include #1424 for this?

aykevl replied:
Done.

QuLogic replied:
Thanks!

aykevl commented:
I included the latest changes, so this release is now ready from my point of view.

aykevl commented:
@deadprogram it appears that TinyHCI is stuck.

deadprogram commented:
OK TinyHCI fixed.

deadprogram commented:
@aykevl this needs to be updated for some of the commits that have landed since it was created.

Are there any other commits that need to merge now before the actual release?

aykevl commented:
Rebased and updated.



> Are there any other commits that need to merge now before the actual release?

I don't think so. And I'd rather get the release out and fix things the next release than delay this release, to be honest.

deadprogram commented:
Agreed! Now merging.
